front/src/modules/ui/input/components/TextInput.tsx

```diff
@@ -45,10 +46,16 @@ const StyledLabel = styled.span`
   text-transform: uppercase;
 `;
 
+const StyledInfo = styled.span`
```

Comment:
Ok! Could you add a story to TextInput.stories.tsx to cover that?

Reply:
ok